test/kubeobjects/namespace/namespace.go

```diff
@@ -68,14 +68,14 @@ func DeleteIfExists(name string) func(ctx context.Context, environmentConfig *en
 	}
 }
 
-func Recreate(name string) func(ctx context.Context, environmentConfig *envconf.Config, t *testing.T) (context.Context, error) {
+func Recreate(name string, labels map[string]string) func(ctx context.Context, environmentConfig *envconf.Config, t *testing.T) (context.Context, error) {
```

Comment:
Instead of having a recreate function which second parameter is nil most of the time, I'd suggest either creating a second function RecreateWithLabels, using a builder pattern for a namespace to pass to a client, or recreate the desired namespace struct directly and pass it to a client.

Reply:
namespace builder pattern used